<!-- -*- mode: XML -*- -->
<launch>
  <rosparam file="$(find mimi_manipulation_pkg)/config/mimi_specification.yaml" command="load" ns="mimi_specification"/> 
  <rosparam file="$(find mimi_manipulation_pkg)/config/object_dict.yaml" command="load" ns="object_dict"/>
  <node pkg="mimi_manipulation_pkg" name="object_recognizer" type="object_recognizer.py" />
  <node pkg="mimi_manipulation_pkg" name="three_dimensional_position_estimator" type="three_dimensional_position_estimator" />
  <!-- <rosparam file="$(find mimi_manipulation_pkg)/config/location_list.yaml" command="load"/> -->
  <node pkg="mimi_manipulation_pkg" name="object_grasper" type="object_grasper.py" />
  <node pkg="mimi_manipulation_pkg" name="manipulation_master" type="manipulation_master.py" />
</launch>
